src/shaderlib/picking/picking.js

```diff
@@ -0,0 +1,95 @@
+const DEFAULT_MODULE_OPTIONS = {
+  pickingSelectedColor: null, //  Set to a picking color to visually highlight that item
+  pickingThreshold: 1.0,
+  pickingActive: false // Set to true when rendering to off-screen "picking" buffer
+};
```

Comment:
Merging this back to luma will be a breaking change then? We'll want to be avoid that if possible.